### ISSUE TITLE
Find in Files does not search scene files

### ISSUE THREAD

**Issue by chucklepie** [godotengine/godot#28285]
**Godot version:**
3.1

**OS/device including version:**
Windows 10

**Issue description:**
Find in files seems to only search within .gd files. If your code is in the .tscn file (i.e. a large majority of the code) then it does not find it.

**Steps to reproduce:**
1. Have some known text in a scene code and a gd file.
2. Use Find In Files.
3. You will not get any results for the text within the scene code.

btw, it would be really nice if you could perform a ctrl/shift/f without having to first open a code file. More a quality of life request than a bug (but you're finding in all files, why the need to have a file open?)

Thanks.

**Comment by sparkart** [comment]
New Godot user here.... I would be interested in working on this issue, although I have a small question. Could you provide details on how to write code that would appear in a .tscn file?

**Comment by Calinou** [comment]
> Could you provide details on how to write code that would appear in a .tscn file?

This is done by creating a new script and making it built-in (instead of saving it to a `.gd` file).

**Comment by neilw1** [comment]
> > Could you provide details on how to write code that would appear in a .tscn file?
> 
> This is done by creating a new script and making it built-in (instead of saving it to a `.gd` file).

Hello,
1. Create a blank node
2. Click on add script
3. In the dialog select 'built in script'

To be honest, it took me ages to figure out how to do it to tell you. This option is now off for me by default, so either it's on by default and I clicked the toggle, or it's off by default and I clicked the wrong toggle :)

The problem I see it, is the tscn file also has a lot of xml for the other parts of the scene, like properties, etc. so it will have to filter these out.

I'm actually now wondering whether having it built in is such a good idea, as all it saves is an extra file? I guess there must be a use case for it. VS, for example, fixes the project tree complexity by nesting the code behind file in a level below each top level 'form'...
